cmd/crio/main.go

```diff
@@ -454,7 +454,12 @@ func main() {
 			logrus.Fatalf("failed to listen: %v", err)
 		}
 
-		s := grpc.NewServer()
+		// The default grpc msg size is 4MB.
+		// We bump it up to 16MB for allowing more containers.
+		maxMsgSize := 16 * 1024 * 1024
+		s := grpc.NewServer(
+			grpc.MaxMsgSize(maxMsgSize),
+		)
```

Comment:
is this anything that someone would want to toggle via crio.conf?

Reply:
We added this in newer branches with crio.conf tunables. We don't want to support that for 1.9.